documentation/troubleshooting.md

```diff
@@ -1,18 +1,18 @@
 # Troubleshooting Guide
 
-Here is our guide for diagnosing specific issues with `Dotnet-Monitor`. This is an ongoing effort to add issues and solutions to this guide and is never complete but you can help us by letting us know in the issues sections if we should add something here
+Here is our guide for diagnosing specific issues with `dotnet monitor`. This is an ongoing effort to add issues and solutions to this guide; you can help us by letting us know in the Issues section if we should add something here.
```

Comment:
Should "Issues section" link to https://github.com/dotnet/dotnet-monitor/issues?